docs/architecture/porting-existing-aspnet-apps/strategies-migrating-in-production.md

```diff
@@ -9,6 +9,17 @@ ms.date: 11/13/2020
 
 ## References
 
+*Notes*
+
+- Strategies for migrating while app is in production
+  - Refactoring models and business logic to .NET Standard libraries
+  - The importance of good test coverage
+  - Extracting front-end assets to shared static content server/cdn
+  - Dividing app into multiple services
+    - Link to existing microservices guidance
+  - Deploying multiple versions or parts of a Web API side-by-side in IIS
+  - Strangler pattern
```

Comment:
Maybe also add multi-targeting and patterns for having separate code paths for Core and Framework (#if, project files that include different source files via globbing patterns like *.core.cs, etc.)